### ISSUE TITLE
stopwords not being ignored in search...

### ISSUE THREAD

**Issue by redlibrarian** [ualbertalib/discovery#490]
though "stopwords_en.txt" is included in the appropriate text fields, I think...

**Comment by redlibrarian** [comment]
Weird though.... maybe this is working. Searching for

canada ottawa

returns the same results as

canada and ottawa

which is as it should be.

**Comment by redlibrarian** [comment]
BUT: searching for

Perestroika New Thinking for Our Country and the World

returns 0 results, while

Perestroika New Thinking for Our Country World

finds the correct book...

**Comment by redlibrarian** [comment]
Another example:

philosophy social sciences 

finds the correct ejournal but

philosophy of the social sciences

does not...

**Comment by pgwillia** [comment]
If you can find the raw Solr query in the logs for these cases, try adding ```debugQuery=true``` and you'll see how the query is parsed and why the documents matched.

**Comment by redlibrarian** [comment]
OK, will do - thanks. In the end, I'm hoping I can figure out why documents are being excluded...

**Comment by redlibrarian** [comment]
So, I reduced the mm value in solrconfig.xml to 0 which, according to the docs, makes it the same as "OR"ing all searched fields, which is the default operator we have specified in schema.xml. This seems to fix the problem, but I have no idea if there will be unintended consequences around this...

"mm" is the "must-match" value, and can be quite esoteric. Essentially, what it does is give a certain value to how many fields much be matched, etc. We had some default blacklight value in there, which I think must have been present from the beginning, so I assume we always had this problem. 

Anyway, I'm going to move this ticket to post-launch so we can keep an eye on it.
